app/components/editor_component/editor_component.html.slim

```diff
@@ -2,12 +2,9 @@
   .editor-component__controls
     label class="#{label_classes}" data-action="click->editor#focus" for="#{@field_name}-field" id=@label[:id]
       = @label[:text]
-    - if @hint.present?
-      .govuk-hint id="#{@field_name}-hint"
-        = @hint
+    .govuk-hint id="#{@field_name}-hint"
+      = @hint.presence || "You can copy and paste any information and keep bullet point formatting."
     .editor-component__toolbar data-editor-target="toolbar"
```

Comment:
Removing this line breaks the input behaviour. So I ended up leaving it while removing the toolbar button only.